bin/generators/plugins/configure.js

```diff
@@ -0,0 +1,142 @@
+const path = require('path');
+const parentRequire = require('parent-require');
+const eg = require('../../eg');
+const PluginInstaller = require('../../../lib/plugin-installer');
+
+module.exports = class extends eg.Generator {
+  constructor (args, opts) {
+    super(args, opts);
+
+    this.installer = null;
+    this.pluginOptions = null;
+
+    this.configureCommand({
+      command: 'configure <package>',
+      description: 'Configure a plugin',
+      builder: yargs =>
```

Comment:
Nit: `(yargs) =>`

Reply:
Agree to disagree.  😃

Reply:
As long we stick with the same convention, I'm fine with that. I saw some inconsistencies around, though.